common/changes/office-ui-fabric-react/vertical-slider-safari_2018-11-13-02-26.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Slider: replace button with div so vertical Sliders render on Safari",
+      "type": "patch"
```

Comment:
Wouldn't changing from `button` to `div` count as a minor change instead of patch? Theoretically this is a breaking change since user could've target the `button` element in styling or even logic. In practice we give them all required className hooks but so the chances to break someone are slim but still exist.

Reply:
This has been a good, perennial question and this falls into a bit of a gray area.

Having said that, our move to merge-styles was precisely to not have the DOM structure be considered part of our API contract so in principle, we wouldn't consider this a change to the API...

Reply:
Ah if so then the change should be AT LEAST a minor... (whenever the API file changes, it would be either a MAJOR (breaking) or MINOR).

Reply:
Updated change type to minor. @dzearing is changing the type of buttonProps from HTMLButtonAttributes to HTMLDivAttributes an acceptable minor change?